pkg/upgrade/v1_22_0_test.go

```diff
@@ -20,12 +20,17 @@ func TestUpgradeJaegerTagssv1_22_0(t *testing.T) {
 		"jaeger.tags": "somekey=somevalue",
 	})
 
+	storageOpts := v1.NewOptions(map[string]interface{}{
+		"server-urls": "https://example:9200",
+	})
+
 	nsn := types.NamespacedName{Name: "my-instance"}
 	existing := v1.NewJaeger(nsn)
 	existing.Status.Version = "1.21.0"
 	existing.Spec.AllInOne.Options = opts
 	existing.Spec.Agent.Options = opts
 	existing.Spec.Collector.Options = opts
+	existing.Spec.Storage.Options = storageOpts
```

Comment:
Add the Ingress as well, given that there was a bug with it too.

Reply:
Done!